web/libs/editor/src/regions/TextAreaRegion/TextAreaRegion.module.scss

```diff
@@ -1,47 +1,69 @@
 .mark {
-  cursor: pointer;
-  background-color: #f6ffed;
-  border: 1px solid #b7eb8f;
-  border-radius: 5px;
-  margin: 0;
-  padding: 0.4em 1em;
+  background-color: var(--surface-surface);
+  border: 1px solid var(--surface-border);
+  border-radius: var(--radius-small);
+  margin: 0 var(--spacing-small) 0 0 !important;
+  padding: var(--spacing-small) var(--spacing-medium);
   white-space: pre-line;
+  box-shadow: 0px 4px 12px 0px var(--black_4), 0px 1px 2px 0px var(--black_10);
+  flex-grow: 1;
 
-  &.selected {
-    border: 1px solid red;
+  &[class*=selected] {
```

Comment:
referencing the class in the js/ts files as like `className={styles.selected}` should remove the need for these kinds of selectors